geode-core/src/main/java/org/apache/geode/cache/client/internal/ProxyRegion.java

```diff
@@ -187,7 +187,7 @@ public void destroyRegion(Object callbackArgument) throws CacheWriterException,
   public Set entries(boolean recursive) {
```

Comment:
I think it is wrong to keep these methods your are removing from Region on classes that implement the Region interface. It is too bad that these impls did not have @Override on them because that would have caused compilation failures.